src/rabbit_nodes_common.erl

```diff
@@ -72,8 +72,7 @@ epmd_port() ->
     end.
 
 ensure_epmd() ->
-    {ok, [[Prog]]} = init:get_argument(progname),
-    Exe = os:find_executable(Prog),
+    {ok, Exe} = rabbit_misc:get_erl_path(),
```

Comment:
This fails to compile in CI:
```
src/rabbit_nodes_common.erl:76: variable 'Prog' is unbound
```

Reply:
That's weird, I ran the `ct-unit` suite locally before pushing the code :thinking:

Reply:
I cannot reproduce and the line does not mention `Prog` any more:

``` erl
Exe = rabbit_runtime:get_erl_path(),
ID = rabbit_misc:random(1000000000),
```